controlplane/kubeadm/internal/filters.go

```diff
@@ -215,8 +216,8 @@ func matchInitOrJoinConfiguration(machineConfig *bootstrapv1.KubeadmConfig, kcp
 	// *Note* This assumes that newly added default values never
 	// introduce a semantic difference to the unset value.
 	// But that is something that is ensured by our API guarantees.
-	bootstrapv1.DefaultKubeadmConfigSpec(kcpConfig)
-	bootstrapv1.DefaultKubeadmConfigSpec(&machineConfig.Spec)
+	util.DefaultKubeadmConfigSpec(kcpConfig)
```

Comment:
Is this the same package others have their defaulting in? Or do we normally construct the defaulting webhook and use that to default the spec?